### PR TITLE
board: ce: Add product files for e850-96

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

**Comment by ric96** [comment]
Documentation bits pending till after merge to preven broken links

**Comment by ric96** [comment]
Need to wait till we get approval from vendor first

**Review by kylekirkby — APPROVED** [review]

**Comment by ric96** [comment]
@kylekirkby please merge, got it approved

**Pull request merged**
